Source/Flow/Private/Nodes/Route/FlowNode_CustomOutput.cpp

```diff
@@ -14,9 +15,44 @@ UFlowNode_CustomOutput::UFlowNode_CustomOutput(const FObjectInitializer& ObjectI
 
 void UFlowNode_CustomOutput::ExecuteInput(const FName& PinName)
 {
-	if (!EventName.IsNone() && GetFlowAsset()->GetCustomOutputs().Contains(EventName) && GetFlowAsset()->GetNodeOwningThisAssetInstance())
+	UFlowAsset* FlowAsset = GetFlowAsset();
+	check(IsValid(FlowAsset));
+
+	if (EventName.IsNone())
```

Comment:
Added some logged warnings for bad configuration or mismatched parameters (would fail silently before).